docs/building.md

```diff
@@ -132,11 +132,13 @@ of the `(i686|x86_64)-w64-mingw32` directory.
 Make sure to install the `C++ CMake tools for Windows` and `Windows SDK` component for Visual Studio.
 *Note: `Windows SDK` component should match your Windows build version.*
 
-Install vcpkg following the instructions from https://github.com/microsoft/vcpkg#quick-start.
-Don't forget to perform _user-wide integration_ step for additional convenience.
+Install vcpkg following the instructions from https://github.com/microsoft/vcpkg#quick-start-windows. Don't forget to
+perform _user-wide integration_ step for additional convenience.
```

Comment:
Avoid adding odd line breaks

Reply:
I tried but my IDE forced it. Give me a few minutes to deactivate this!